Not building on SmartOS and node 0.10.x

```
> usage@0.3.8 install /root/node_modules/usage
> node-gyp rebuild

make: Entering directory `/root/node_modules/usage/build'
  CXX(target) Release/obj.target/sysinfo/src/binding.o
  CXX(target) Release/obj.target/sysinfo/src/solaris.o
../src/solaris.cpp: In function 'int get_usage(int, ps_usage_t*)':
../src/solaris.cpp:52:61: warning: deprecated conversion from string constant to 'char*' [-Wwrite-strings]
../src/solaris.cpp: In function 'v8::Handle<v8::Value> GetUsage(const v8::Arguments&)':
../src/solaris.cpp:122:76: error: invalid conversion from 'void (*)(uv_work_t*) {aka void (*)(uv_work_s*)}' to 'uv_after_work_cb {aka void (*)(uv_work_s*, int)}' [-fpermissive]
In file included from /root/.node-gyp/0.10.13/src/node.h:61:0,
                 from ../src/solaris.h:7,
                 from ../src/solaris.cpp:2:
/root/.node-gyp/0.10.13/deps/uv/include/uv.h:1432:15: error:   initializing argument 4 of 'int uv_queue_work(uv_loop_t*, uv_work_t*, uv_work_cb, uv_after_work_cb)' [-fpermissive]
make: *** [Release/obj.target/sysinfo/src/solaris.o] Error 1
make: Leaving directory `/root/node_modules/usage/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/opt/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:789:12)
gyp ERR! System SunOS 5.11
gyp ERR! command "node" "/opt/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /root/node_modules/usage
gyp ERR! node -v v0.10.13
gyp ERR! node-gyp -v v0.10.6
gyp ERR! not ok 
npm ERR! weird error 1

```

--- a/src/solaris.cpp
+++ b/src/solaris.cpp
@@ -119,7 +119,7 @@ Handle<Value> GetUsage(const Arguments& args) {
     uv_work_t* req = new uv_work_t();
     req->data = usageData;
 
-    uv_queue_work(uv_default_loop(), req, AsyncGetUsage, AsyncAfterGetUsage);
+    uv_queue_work(uv_default_loop(), req, AsyncGetUsage, (uv_after_work_cb)AsyncAfterGetUsage);
     
     return scope.Close(Undefined());
 }

